### PR TITLE
UIREQ-382 - FE: Print pick slips for service point on demand

### PR DESCRIPTION
**Pull request by maximdidenkoepam**
# Purpose
Get all pick slips in the queue that are associated with current service point to print

# Link
https://issues.folio.org/browse/UIREQ-382

# Screenshots
Screen Shot 2019-11-27 at 16 25 23
Screen Shot 2019-11-27 at 16 25 33

### CODE REVIEW

File: src/components/PrintButton/PrintButton.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          content={this.getContent}
+          removeAfterPrint
+          trigger={this.renderTriggerButton}
+          onAfterPrint={onAfterPrint}
+          onBeforePrint={onBeforePrint}
+        />
+        <div className={css.hiddenContent}>
+          <div ref={this.printContentRef}>
+            {dataSource.map(source => {
+              return (
```
Comment (mkuklis) [inline src/components/PrintButton/PrintButton.js]: @maximdidenkoepam I wonder if we need the `return` here. Could we just get rid of `{`.

File: src/routes/RequestsRoute.js [hunk 2]
```diff
@@ -173,6 +177,12 @@ class RequestsRoute extends React.Component {
       path: 'cancellation-reason-storage/cancellation-reasons',
       records: 'cancellationReasons',
     },
+    staffSlips: {
```
Comment (marcjohnson-kint) [inline src/routes/RequestsRoute.js]: Is this a new backend interface dependency, it appears to be a new endpoint for this module, and there doesn’t appear to be an entry in the package.json for it?

File: src/routes/RequestsRoute.js [hunk 3]
```diff
@@ -173,6 +177,12 @@ class RequestsRoute extends React.Component {
...
+      type: 'okapi',
+      records: 'staffSlips',
+      path: 'staff-slips-storage/staff-slips?limit=100',
```
Comment (marcjohnson-kint) [inline src/routes/RequestsRoute.js]: Also, I was of the understanding that new slip templates weren’t going to use the older staff slips templates and instead use the template engine, did I misunderstand that?

File: src/routes/RequestsRoute.js (hunk 2 above)
Comment (zburke) [reply src/routes/RequestsRoute.js]: I agree, @marcjohnson-kint ; it looks like we need to add a dependency on mod-circulation-storage.

**Pull request merged**